lib/content-services/document-list/services/document-actions.service.ts

```diff
@@ -101,10 +101,7 @@ export class DocumentActionsService {
     private prepareHandlers(actionObservable, type: string, action: string, target?: any, permission?: string): void {
         actionObservable.subscribe(
             (fileOperationMessage) => {
-                if (target && typeof target.reload === 'function') {
```

Comment:
can we create a method with those instructions?

Reply:
Just a note: I think we will need a second review later on when introducing multi-selection actions.